CONTRIBUTING.md

```diff
@@ -1,91 +1,101 @@
 
-Bisq is not a company, but operates as a _decentralized autonomous organization_ (DAO). For any work merged into Bisq's `master` branch, you can [submit a compensation request](https://docs.bisq.network/dao/phase-zero.html#how-to-request-compensation) and earn BSQ (the Bisq DAO native token). Learn more about the Bisq DAO and BSQ [here](https://docs.bisq.network/dao/phase-zero.html).
+Go on and do the actual work.
 
 
-## Style and Coding Conventions
+# Pull request rules
 
-### Configure Git user name and email metadata
+Please honor the list of rules below for your pull request to get merged. Due to very limited resources, we are forced to enforce these quite rigorously.
 
-See https://help.github.com/articles/setting-your-username-in-git/ for instructions.
+## Honor Git best practice
+ - Create a topic branch off master
+ - Commit little, commit often
+ - Maintain a clean commit history (no merges commits!, use rebase and force-push if necessary)
+ - Use meaningful commit subjects AND description (then, now, good, bad, risks)
+ - Do so in a well-formed manner (from https://chris.beams.io/posts/git-commit/#seven-rules)
+   1. Separate subject from body with a blank line
+   2. Limit the subject line to 50 characters
+   3. Capitalize the subject line
+   4. Do not end the subject line with a period
+   5. Use the imperative mood in the subject line
+   6. Wrap the body at 72 characters
+   7. Use the body to explain what and why vs. how
 
-### Write well-formed commit messages
+## Respect Bisq's coding style
+ - The .editorconfig settings in this repository ensure consistent management of whitespace, line endings and more. Most modern editors support it natively or with plugin. See http://editorconfig.org for details. See also bisq-network/style#10.
+ - Also, please honor the issues in the [bisq-network/style](https://github.com/bisq-network/style/issues) repository
+ - Do not commit import reorgs or any other non-changes!
 
-From https://chris.beams.io/posts/git-commit/#seven-rules:
 
- 1. Separate subject from body with a blank line
- 2. Limit the subject line to 50 characters
- 3. Capitalize the subject line
- 4. Do not end the subject line with a period
- 5. Use the imperative mood in the subject line
- 6. Wrap the body at 72 characters
- 7. Use the body to explain what and why vs. how
+## Test your code!
+ - Create automated tests for your work. JUnit is available. If you are fixing a bug, create a test that demonstrates the bug by failing. Then fix the bug.
+ - If automated testing is not feasible (we know it can be hard - working on it ...), provide a textual testing procedure (which we can include in our manual testing checklist): setup, prerequisits, steps, expected results
+ - If that is not possible either, provide some info (numbers, screenshots, testing procedure) on how you tested your stuff
+ - If all is lost, state why you have not been able to do the above.
```

Comment:
I like this section. The burden is on the contributor to convince the reviewers that the code is solid. Not the other way around.